src/app/settings/components/repositoryLocationListItem.tsx

```diff
@@ -63,7 +64,7 @@ export default class RepositoryLocationListItem extends React.Component<Reposito
                         </div>
                         {item.repositoryLocationType !== REPOSITORY_LOCATION_TYPE.Public && <IconButton
                             className="remove-button"
-                            iconProps={{ iconName: 'cancel' }}
+                            iconProps={{ iconName: CANCEL }}
```

Comment:
I have a couple more tests on my machine we can add to the spec file for this component.